src/index.js

```diff
@@ -16,3 +18,26 @@ for (let i = 1; i <= list.length; i += 1) {
     }
   });
 }
+
+clear.addEventListener('click', () => {
+  const newlist = list.filter((element) => element.completed === false);
+  list.forEach((element) => {
+    if (!newlist.includes(element)) {
+      list.splice(list.indexOf(element), 1);
+    }
+  });
+  let i = 1;
+  list.forEach((element) => {
+    element.index = i;
+    i += 1;
```

Comment:
✅ A function for the "Clear all completed" button using the Array.filter() method is implemented 👍🏻 

- [ ] [REQUIRED] The function does not work well yet. When I check all the tasks to be cleared, some are left in the list which shouldn't be the case.

| Before | After |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/171865508-6f5be1ab-7d0d-4ce3-b560-94826431052d.png) | ![image](https://user-images.githubusercontent.com/4247740/171865609-4566444a-a521-4cf4-b00b-0db78e239cc0.png) |